bindings/python/src/lib.rs

```diff
@@ -35,6 +35,7 @@ type MemcpyFn = unsafe extern "C" fn(
 static TORCH_MODULE: GILOnceCell<Py<PyModule>> = GILOnceCell::new();
 static NUMPY_MODULE: GILOnceCell<Py<PyModule>> = GILOnceCell::new();
 static TENSORFLOW_MODULE: GILOnceCell<Py<PyModule>> = GILOnceCell::new();
+static PADDLE_MODULE: GILOnceCell<Py<PyModule>> = GILOnceCell::new();
```

Comment:
i remove these  modifications and i got some error. I will remove `test_deserialization_safe_open` test
![image](https://user-images.githubusercontent.com/50394665/217236030-faaf9518-9e76-4873-b3d5-6bf481e7aa62.png)

Reply:
> we need to remove those modifications. The rust codebase won't create those tensors directly, so just like TF and Flax, we don't open those either.

Hello,  since ` The rust codebase won't create those tensors directly`, it there another way to support `safe_open` for paddle backend ?

Reply:
If there is a *python* only way to read a safetensors file, we could use that in the rust code  (the binding can just call the equivalent Python code).

This is doable only if the tensor creation is purely lazy so it's still compatible with the slice loading.